programs/multisig/src/lib.rs

```diff
@@ -239,6 +252,20 @@ pub struct ExecuteTransaction<'info> {
     transaction: ProgramAccount<'info, Transaction>,
 }
 
+#[derive(Accounts)]
+pub struct DropTransaction<'info> {
+    multisig: ProgramAccount<'info, Multisig>,
+    #[account(
+        mut,
+        has_one = multisig,
+        constraint = transaction.did_execute == true,
+        constraint = transaction.successor == Some(*successor.key),
+    )]
+    transaction: ProgramAccount<'info, Transaction>,
+    #[account(mut)]
+    successor: AccountInfo<'info>,
```

Comment:
should the successor sign this?

Reply:
I do not think that this is required. Fewer signatures are less transaction fee :zany_face:

Reply:
roger that